polkadot/runtime/rococo/constants/src/lib.rs

```diff
@@ -37,14 +37,15 @@ pub mod currency {
 
 /// Time and blocks.
 pub mod time {
+	use runtime_common::prod_or_fast;
+
 	use primitives::{BlockNumber, Moment};
 	pub const MILLISECS_PER_BLOCK: Moment = 6000;
 	pub const SLOT_DURATION: Moment = MILLISECS_PER_BLOCK;
 
 	frame_support::parameter_types! {
-		pub storage EpochDurationInBlocks: BlockNumber = option_env!("ROCOCO_EPOCH_DURATION")
-			.map(|s| s.parse().expect("`ROCOCO_EPOCH_DURATION` is not a valid `BlockNumber`"))
-			.unwrap_or(1 * MINUTES);
+		pub EpochDurationInBlocks: BlockNumber =
+			prod_or_fast!(1 * HOURS, 1 * MINUTES, "ROCOCO_EPOCH_DURATION");
```

Comment:
Should it not rather check for feature `fast-runtime` here?

Reply:
https://github.com/paritytech/polkadot-sdk/blob/50390950d87fb745ec5ff46be698cac93113efb7/polkadot/runtime/common/src/lib.rs#L263-L279